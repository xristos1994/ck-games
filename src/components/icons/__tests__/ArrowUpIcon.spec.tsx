import React from 'react';
import { ArrowUpIcon } from './../';
import renderer from 'react-test-renderer';

describe('components/icons/ArrowUpIcon', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<ArrowUpIcon />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
