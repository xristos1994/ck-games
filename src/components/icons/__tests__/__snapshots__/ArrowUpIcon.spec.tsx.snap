// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/icons/ArrowUpIcon renders correctly 1`] = `
<svg
  height="100%"
  version="1.1"
  viewBox="0 0 512 512"
  width="100%"
  xmlns="http://www.w3.org/2000/svg"
>
  <polygon
    fill="currentColor"
    points="396.6,352 416,331.3 256,160 96,331.3 115.3,352 256,201.5"
  />
</svg>
`;
